src/Avalonia.Input/Raw/RawPointerEventArgs.cs

```diff
@@ -14,6 +14,10 @@ public enum RawPointerEventType
         RightButtonUp,
         MiddleButtonDown,
         MiddleButtonUp,
+        XButton1Down,
```

Comment:
Same here, values starting from `Move` will have old values in dependent assemblies compiled before this change